pkg/manifest/manifest.go

```diff
@@ -5,38 +5,36 @@
 package manifest
 
 import (
+	"encoding"
 	"errors"
 	"net/http"
 
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
+// ErrNotFound is returned when an Entry is not found in the manifest
```

Comment:
I see that this package does not follow the full sentence comment guideline (missing dots at the end). I would suggest to have that in order to have better documentation in godoc. https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences referenced in https://github.com/ethersphere/bee/blob/master/CODING.md.

This is also more related to the current state of the code in master, than to this PR.

Reply:
shoot, i missed this.

thanks, fixed